geomesa-gt/geomesa-gt-partitioning/src/main/scala/org/locationtech/geomesa/gt/partition/postgis/dialect/PartitionedPostgisDialect.scala

```diff
@@ -418,6 +418,9 @@ object PartitionedPostgisDialect {
     val WriteAheadPartitionsTableSpace = "pg.partitions.tablespace.wa-partitions"
     val MainTableSpace = "pg.partitions.tablespace.main"
 
+    // set postgres table wal logging
+    val WalLogEnabled = "pg.wal.log.enabled"
```

Comment:
`wal log` is redundant (write ahead log log), could we change this to `pg.wa.log.enabled` or `pg.wal.enabled`?

Reply:
changed to `pg.wal.enabled`

8784b23a727736800eb4e7451e95fa162108bf21